SYCL/Reduction/reduction_range_1d_rw.cpp

```diff
@@ -20,29 +30,40 @@ int main() {
 
   constexpr access::mode RW = access::mode::read_write;
   // Fast-reduce and Fast-atomics. Try various range types/sizes.
-  test<class A1, int>(Q, 0, 99, std::plus<int>{}, range<1>(1));
-  test<class A2, int>(Q, 0, 99, std::plus<>{}, range<1>(2));
-  test<class A3, int>(Q, 0, 99, std::plus<>{}, range<1>(7));
-  test<class A4, int>(Q, 0, 99, std::plus<>{}, range<1>(64));
-  test<class A5, int>(Q, 0, 99, std::plus<>{}, range<1>(MaxWGSize * 2));
-  test<class A6, int>(Q, 0, 99, std::plus<>{}, range<1>(MaxWGSize * 2 + 5));
+  tests<class A1, int>(Q, 0, 99, std::plus<int>{}, range<1>(1));
```

Comment:
How did it work with `test` ? Were errors ignored?

Reply:
It looks like the results were indeed being ignored. They shouldn't be anymore though.